datagrepper/app.py

```diff
@@ -96,6 +96,13 @@ def inject_variable():
     return extras
 
 
+@app.after_request
+def add_cors(response):
+    """Allow CORS for all URLs."""
+    response.headers['Access-Control-Allow-Origin'] = '*'
```

Comment:
Configuration to turn it on/off, or something more sophisticated? If we're going to do anything more than on/off, we should probably instead use something like:

http://flask-cors.readthedocs.io/en/latest/

I didn't take that approach because I wanted to avoid adding a new dependency, and it seemed consistent with the pre-CORS behavior (everything is open to everyone). I'm happy to revisit those assumptions though.

Reply:
We could just take the `'*'` string from config... ?  (plus the ability to turn it off if the config value is `None`).

Reply:
Yeah I was thinking along the lines of @ralphbean , have a configuration key, if set to ``None`` skip, otherwise take the value of that configuration key (for example ``'*'``)